cloudigrade/util/tests/test_aws.py

```diff
@@ -28,9 +28,13 @@ def test_extract_account_id_from_arn(self):
 
     def test_error_extract_account_id_from_invalid_arn(self):
         """Assert error in account ID extraction from a badly-formed ARN."""
-        mock_arn = faker.Faker().text()
-        with self.assertRaises(Exception):  # TODO more specific exceptions
-            aws.extract_account_id_from_arn(mock_arn)
+        bad_index_arn = 'The quick brown fox jumps over the lazy dog.'
+        empty_arn = '::::'
+
+        with self.assertRaises(IndexError):
```

Comment:
The intent of the `TODO` was actually to implement *our own* more specific exception so we could intelligently catch and present situation-appropriate messaging later, e.g. `InvalidArnException`. I guess that can wait for later when/if we do more user-friendly error handling, though, since it hasn't been needed yet.

Reply:
So that was my original concern with that `TODO`, it simply did not have enough context, hence I felt it was "useless". I'm more than happy to rollback the `todo` if you wish though.

Reply:
Fair point. :) I was used to previous teams I'd worked with recognizing "more specific exceptions" as the shorthand cue for implementing our own, but you're right that I should have been more specific here with my intentions.

Yeah, let's put that `TODO` back as something like
> TODO implement our own situation-appropriate exception here, e.g. `InvalidArnException`